### DISCUSSION TITLE
Selecting an custom character in debug menu, will crash my game and give me this error

### DISCUSSION THREAD

**Discussion by DennisAnabones** [KadeArchive/Kade-Engine#238] [Q&A]
![Knipsel31](https://user-images.githubusercontent.com/82602747/115123137-f9962700-9fbb-11eb-99dc-e969d60f463b.PNG)
It says: haxe.xml expected node line 2 char 2, any way to fix it?

**Comment by theDetourist** [comment]
What steps did you take to create that character?

**Reply by DennisAnabones** [reply]
Made the spritesheet with Animate, and wrote it down. Heres the source code
![Knipsel12](https://user-images.githubusercontent.com/82602747/115127237-49cdb300-9fd5-11eb-914f-5249dd3ae062.PNG)

**Reply by theDetourist** [reply]
```case``` needs a fully opened and closed body:

```
case "something":
{
    (code)
}
```

Also, how does your custom characters's ```.xml``` file looks like? (If possible, drop it @ [Pastebin](https://pastebin.com/), please)

**Reply by DennisAnabones** [reply]
also i tried bracketing it but it gave me an: source/Character.hx:542: characters 2-10 : Expected } error

**Reply by theDetourist** [reply]
Gonna need your Character.hx too then, it means something in the code's not closed properly.

Nvm: You don't have to do it. It should be working just fine code wise without doing that.

**Reply by theDetourist** [reply]
Are you compiling using the ```-debug``` flag? If not, try that and show to us what's there when it crashes.

**Reply by DennisAnabones** [reply]
Doesn't work in debug either, So I'm sending you my character.hx file, https://fromsmash.com/Characterhx

**Comment by DennisAnabones** [comment]
https://pastebin.com/sLhEYJZT
